### DISCUSSION TITLE
hexIds in H3ClusterLayer

### DISCUSSION THREAD

**Discussion by brianshano** [visgl/deck.gl#6290] [Q&A]
I'm trying to figure out what the hexIds in a H3ClusterLayer are and how they are related to coordinates?
I have a dataset where geo points are clustered into regions. I don't have boundaries of the regions just a cluster of points where I want to visually highlight the region. 
It looks like H3ClusterLayer is what I want but I can't figure out what the hexId values are??

https://deck.gl/docs/api-reference/geo-layers/h3-cluster-layer
from the docs example: json of one region 
```
{
  mean: 73.333,
  count: 440,
  hexIds: [
    "88283082b9fffff",
    "88283082b1fffff",
    "88283082b5fffff",
    "88283082b7fffff",
    "88283082bbfffff",
    "882830876dfffff"
  ]
},
```

**Comment by kylebarron** [comment]
> The H3ClusterLayer renders regions represented by hexagon sets from the [H3](https://h3geo.org/) geospatial indexing system

I believe that to use this layer your data must already be in H3 hexagons. You can use [`geoToH3`](https://h3geo.org/docs/api/indexing#geotoh3) to convert latitude-longitude points into their H3 cells.

You could also use something like the [`HeatmapLayer`](https://deck.gl/docs/api-reference/aggregation-layers/heatmap-layer) to aggregate points directly, if you don't care about using the H3 system.
